airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -245,7 +245,7 @@ def get_link(
         :return: url link
         """
         if ti_key:
-            flow_id = XCom.get_one(key="return_value", ti_key=ti_key)
+            flow_id = XCom.get_value(key="return_value", ti_key=ti_key)
```

Comment:
We can't use this new method :/ The providers need to continue to work with Airflow 2.1 :/

Reply:
Gah, no. This is the compat shim that _I_ wrote. Ingore me, this is fine.